src/Microsoft.AspNet.Authentication.Facebook/FacebookAppBuilderExtensions.cs

```diff
@@ -18,10 +17,22 @@ public static class FacebookAppBuilderExtensions
         /// </summary>
         /// <param name="app">The <see cref="IApplicationBuilder"/> passed to the configure method.</param>
         /// <returns>The updated <see cref="IApplicationBuilder"/>.</returns>
-        public static IApplicationBuilder UseFacebookAuthentication([NotNull] this IApplicationBuilder app, Action<FacebookOptions> configureOptions = null)
+        public static IApplicationBuilder UseFacebookAuthentication([NotNull] this IApplicationBuilder app, FacebookOptions options)
         {
-            return app.UseMiddleware<FacebookMiddleware>(
-                 new ConfigureOptions<FacebookOptions>(configureOptions ?? (o => { })));
+            return app.UseMiddleware<FacebookMiddleware>(options);
+        }
+
+        /// <summary>
+        /// Authenticate users using Facebook.
+        /// </summary>
+        /// <param name="app">The <see cref="IApplicationBuilder"/> passed to the configure method.</param>
+        /// <param name="configureOptions">Configures the options.</param>
+        /// <returns>The updated <see cref="IApplicationBuilder"/>.</returns>
+        public static IApplicationBuilder UseFacebookAuthentication([NotNull] this IApplicationBuilder app, Action<FacebookOptions> configureOptions)
```

Comment:
I'd rather null guard the configureOptions, which I was doing everywhere else, but yeah I missed this one :)